README.md

```diff
@@ -55,7 +66,7 @@ use std::str::FromStr;
 fn main() -> Result<(), String> {
     let edn_str = "{:a \"2\" :b [true false] :c #{:A {:a :b} nil}}";
     // std::str::FromStr
-    let edn = Edn::from_str(edn_str);
+    let edn :Edn = Edn::from_str(edn_str);
```

Comment:
This isn't needed and the formatting is wrong.

Reply:
it was intentional to add `:Edn`. I got confused with this line